shared/aux.go

```diff
@@ -480,6 +480,21 @@ func appendNodeIfMissing(slice []Node, i Node) []Node {
 	}
 	return append(slice, i)
 }
+
 func EncloseSqBraces(ip string) string {
 	return "[" + ip + "]"
 }
+
+// PrintGetAll prints the output of kubectl get all -A -o wide and kubectl get nodes -o wide
+func PrintGetAll() {
```

Comment:
I think we have this already with `PrintClusterState`

Reply:
```
	cmd := "kubectl get nodes,pods -A -o wide --kubeconfig=" + KubeConfigFile
```
PrintClusterState is only print that, not all other  resources